instance/tests/api/test_instance.py

```diff
@@ -323,3 +324,53 @@ def test_set_notes_instance_gives_error_if_extra_field_is_provided(self, mock_co
         self.assertFalse(mock_is_valid.called)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
         self.assertEqual({"error": "Instance attributes are not valid."}, response.data)
+
+
+@ddt.ddt
+@patch(
+    'instance.tests.models.factories.openedx_instance.OpenEdXInstance._write_metadata_to_consul',
+    return_value=(1, True)
+)
```

Comment:
We can probably remove these decorators since we're not using ddt or instance factories.